### DISCUSSION TITLE
Urgent Assistance Required for GitHub Account

### DISCUSSION THREAD

**Discussion by hamzaali15** [community/community#135737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Everyone,

I am writing on behalf of my colleague ([@ahmadhuss](https://github.com/ahmadhuss)), whose GitHub account was flagged three months ago. He has been an active contributor to the open-source community for nearly **9 years** and has invested significant time and effort in various projects. However, this is the first time he has encountered such an issue.

The suspension is severely affecting our organization's private repository projects. He submitted a support ticket (#2900690) two months ago, but unfortunately, he has not received any response yet. Additionally, his profile seems to be returning a 404 error and is not visible to the public.

I kindly request your assistance in resolving this matter, as it is critical to our ongoing work and how much time the GitHub support team typically needs to address such issues?

Thanks

**Comment by Budalebah** [comment]
Same here with ticket : https://support.github.com/ticket/2939983

**Comment by prabhjotschugh** [comment]


**Reply by haloxtradee** [reply]
there isnt a twitter account for github support

**Reply by still-hash-ir** [reply]
I am facing the same issue. I don't know why people suggest options without first looking that they exist or not. @prabhjotschugh There is no Twitter account **@githubsupport** and the only way to follow up on the existing ticket is to reply to the email thread. Finally, a new ticket with the same details cannot be created. You will see the **This appears to be a duplicate request** message. I am pretty sure OP and others with their account suspended have already tried these options.

**Reply by haloxtradee** [reply]
Because their unique aim is to get interaction here, weird.

**Reply by haloxtradee** [reply]
And this guy only spamming in many places

**Reply by still-hash-ir** [reply]
Seems like he deleted his comment. What a waste!

**Reply by haloxtradee** [reply]
Yes he deleted, he should be flagged not us! Suppoer team when you think to help us???

**Comment by haloxtradee** [comment]
@budalebah needs same help please

**Comment by prabhjotschugh** [comment]


**Reply by hamzaali15** [reply]
We can't send a direct email to the support team, and they also do not have an X account. The only way to communicate with them is by creating a support ticket, which we have already done as mentioned in the reference [@ahmadhuss](https://github.com/ahmadhuss) (#2900690). However, the problem is that we have not received any feedback on the associated ticket from their support team. The situation would be the same for any duplicate ticket.

Hi Lili, @queenofcorgis, could you please help us understand how long we might have to wait to get feedback on the [@ahmadhuss](https://github.com/ahmadhuss) (#2900690) ticket?

Thanks

**Reply by haloxtradee** [reply]
Yes but support never response, i need urgent help , i have pro but cant send mail to them

**Comment by still-hash-ir** [comment]


**Reply by haloxtradee** [reply]
Bro, no way, they dont care about us and our projects... budalebah is stüll flagged and my all works gone

**Reply by still-hash-ir** [reply]
Deleted the petition comment here. I don't think its needed anymore

**Comment by ahmadhuss** [comment, marked as answer]
Hi Everyone,

I noticed that my account has been functioning normally today. Thank you all for addressing this issue.

Kind regards,
Ahmad Hussnain

**Reply by haloxtradee** [reply]
Oh really? How many days it took? We need also this help

**Reply by still-hash-ir** [reply]
My account has also been restored. Support said that their team is currently experiencing a higher volume of inquiries than usual, which has resulted in longer response times. They have been investigating suspicious activity and account hijacking. As a security measure, they have placed restrictions on some accounts to combat this. Don't lose hope, it took them some time but they are reviewing every support ticket. Hopefully, you will get access too!

**Reply by Budalebah** [reply]
I hope bro ı hope

**Reply by haloxtradee** [reply]
i need same thing pls

**Comment by shreya-sakshi** [comment]
Dear GitHub Support Team,
I recently received a notification stating that my account has been suspended. I am unsure of the reason for this action, as I strive to adhere to GitHub's Terms of Service. Could you please provide more details about the suspension? I would like to resolve this issue as soon as possible.
I am facing the same issue with my account
